libfreerdp/crypto/test/TestFuzzCryptoCertificateReadServerCertificate.c

```diff
@@ -0,0 +1,9 @@
+#include <freerdp/crypto/crypto.h>
+
+extern int LLVMFuzzerTestOneInput(const uint8_t* Data, size_t Size)
+{
+	rdpCertificate* certificate = NULL;
+	certificate_read_server_certificate(certificate, (BYTE*)Data, Size);
```

Comment:
Return value check?

Reply:
What for should we check return value here? The test is for checking reliability, not correctness.

Reply:
What is a right way to include header with `certificate_read_server_certificate`?